src/components/molecules/VenueRoomItem/VenueRoomItem.tsx

```diff
@@ -80,7 +80,10 @@ export const VenueRoomItem: React.FC<VenueRoomItemProps> = ({
     // TS doesn't work properly with const statements and won't 'know' that this is already checked.
     // That's why this is inline instead of isVenuePortal
     if (template !== RoomTemplate.external) {
-      const venueData = buildEmptyVenue(roomValues.venueName, template);
+      const venueData = {
+        ...buildEmptyVenue(roomValues.venueName, template),
```

Comment:
To retain consistency you can pass `parentId` to `buildEmptyVenue` as it seems like always adding `parentId` is a new "norm.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2312/commits/0023b3b4d849d0af4ae51d5af3ca558495cb927e